scripting/GFL-UserManagement.sp

```diff
@@ -140,23 +140,25 @@ public void OnClientDisconnect(int client) {
 }
 
 public void OnClientAuthorized(int client, const char[] sAuth2) {
-	// Get their Steam ID 64.
-	char steamID64[64];
-	GetClientAuthId(client, AuthId_SteamID64, steamID64, sizeof(steamID64), true);
+	if (!IsFakeClient(client)) {
```

Comment:
Imagine not early exiting `if (IsFakeClient(client)) return;`!!! Think of all the whitespace!!!!!!!!!!!!!!!!

Reply:
+1

Reply:
I am a whitespace enjoyer and you will enjoy it too